go/builder.go

```diff
@@ -284,6 +284,31 @@ func (b *Builder) PrependUOffsetT(off UOffsetT) {
 	b.PlaceUOffsetT(off2)
 }
 
+// StringsVector  a shortcut to pack []string
+func (b *Builder) StringsVector(str ...string) UOffsetT {
+	l := len(str)
+	if l == 0 {
+		return UOffsetT(0)
+	}
+	namesOffsets := make([]UOffsetT, l)
+	for j := l - 1; j >= 0; j-- {
+		namesOffsets[j] = b.CreateString(str[j])
+	}
+	return b.VectorArray(namesOffsets...)
+}
+
+// VectorArray  shortcut for vector array ( UOffset array ) like strings / tables / unions
+func (b *Builder) VectorArray(off ...UOffsetT) UOffsetT {
+	if len(off) == 0 {
+		return UOffsetT(0)
+	}
+	b.StartVector(4, len(off), 4)
```

Comment:
Should `4` be the named constant instead? I believe we already define the proper constant elsewhere.

Reply:
ok, i fix via constant.